README.md

```diff
@@ -2,6 +2,36 @@
 
 This repo implements the CSI spec v1.1.0 for [GIG.tech's OpenvCloud](https://gig.tech)
 
+## Ansible role CSI-Driver
+
+OVC CSI driver can be installed with Ansible role [csi-driver](roles/scr-driver)
+
+Role variables:
+
+``` yaml
```

Comment:
Maybe add full path of this file so people know exactly where to look for right away,
`roles/csi-driver/vars/main.yaml`?